db/database.go

```diff
@@ -48,6 +48,16 @@ const (
 	DBCompactRunning
 )
 
+const (
+	ImportEvent uint32 = iota
+	SGWriteEvent
+	BlipWriteEvent
+)
+
+type DocUpdateEvent struct {
```

Comment:
What else do you anticipate putting in this struct, other than type?

Reply:
I was probably thinking too far ahead, but was thinking we may need to be able to pass down the stack a flag to to indicate if a doc mutation/creation has arrived to sync gateway from a non version vector aware peer/client? So initially thought this could be a vehicle to add that flag into and pass down the stack to help handle that type of 'doc update event' in sync gateway.

But you raise a good point i'll update the PR to remove the struct and have it as a variable instead!